sdk/tables/Azure.Data.Tables/src/autorest.md

```diff
@@ -8,7 +8,9 @@ Run `dotnet msbuild /t:GenerateCode` to generate code.
 ``` yaml
 title: Azure.Data.Tables
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/33c52e4f87f3ae21611d45f34db65f9ccc510ea6/specification/cosmos-db/data-plane/Microsoft.Tables/preview/2019-02-02/table.json
+    - $(this-folder)/swagger/table.json
```

Comment:
This is pain.  You are signing up for a world of pain.  What's the plan to undo this as soon as possible?

Reply:
[This](https://github.com/Azure/azure-rest-api-specs/pull/10240) is the primary plan, and local swagger transforms would be the backup plan. I did it this way initially to validate the swagger changes for the PR, but this won't stay long term.